Week1/homework/exercise4.js

```diff
@@ -0,0 +1,6 @@
+var myString = ("Aline Amaral Farah Pereira")
+console.log ("The string is my name")
+console.log (myString)
+var myString = ("This is not my name anymore")
```

Comment:
Couple things similar to before:
- You should avoid using `var` keyword and use `let` or `const` instead
- You need not re-declare a variable once it has been created. Thus, using `var` on line 4 isn't required.